package.json

```diff
@@ -25,6 +25,7 @@
 		"axios": "^0.26.1",
 		"core-js": "^3.8.3",
 		"floating-vue": "^2.0.0-beta.15",
+		"nodemon": "^2.0.15",
```

Comment:
Please rollback this file. I was just trying something but it didn't work out. Please rollback `package.json` and `package-lock.json` as well as the `vue.config.js`